app/Authentication/src/Http/Responses/PasswordResetResponse.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Authentication\Http\Responses;
+
+use Illuminate\Contracts\Support\Responsable;
+use Illuminate\Validation\ValidationException;
+
+class PasswordResetResponse implements Responsable
+{
+    /**
+     * The response status language key.
+     *
+     * @var string
+     */
+    protected $status;
+
+    /**
+     * Create a new response instance.
+     *
+     * @param  string  $status
+     * @return void
+     */
+    public function __construct(string $status)
+    {
+        $this->status = $status;
+    }
+
+    /**
+     * Create an HTTP response that represents the object.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function toResponse($request)
+    {
+        if ($request->wantsJson()) {
+            throw ValidationException::withMessages([
+                'email' => [trans($this->status)],
+            ]);
+        }
+
+        return back()
```

Comment:
This is not needed since it's an API only project. The line above can be the returned result.